ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/CaseMapper.java

```diff
@@ -134,4 +138,16 @@ public Claim from(CCDCase ccdCase) {
 
         return builder.build();
     }
+
+    private YesNoOption convertCCDYesNo(CCDYesNoOption ccdYesNoOption) {
+        return Optional.ofNullable(ccdYesNoOption)
+            .map(e -> EnumUtils.getEnumIgnoreCase(YesNoOption.class, e.name()))
+            .orElse(null);
```

Comment:
if it is null, should it be defaulted to false?

Reply:
No. It should default to null. null != NO. Null means no decision has been selected, NO means no has been selected.